homeassistant/components/honeywell/config_flow.py

```diff
@@ -53,3 +60,38 @@ async def async_step_import(self, import_data):
                 CONF_HEAT_AWAY_TEMPERATURE: import_data[CONF_HEAT_AWAY_TEMPERATURE],
             }
         )
+
+    @staticmethod
+    @callback
+    def async_get_options_flow(config_entry):
+        """Options callback for Honeywell."""
+        return HoneywellOptionsFlowHandler(config_entry)
+
+
+class HoneywellOptionsFlowHandler(config_entries.OptionsFlow):
+    """Config flow options for Honeywell."""
+
+    def __init__(self, entry: config_entries.ConfigEntry) -> None:
+        """Initialize Honeywell options flow."""
+        self.config_entry = entry
+
+    async def async_step_init(self, user_input=None):
+        """Manage the options."""
+        if user_input is not None:
+            return self.async_create_entry(title=DOMAIN, data=user_input)
+
+        return self.async_show_form(
+            step_id="init",
+            data_schema=vol.Schema(
+                {
+                    vol.Required(
+                        CONF_COOL_AWAY_TEMPERATURE,
+                        default=DEFAULT_COOL_AWAY_TEMPERATURE,
```

Comment:
if should default to the previously configured value, no ?

Reply:
@rdfurman can you please confirm this for @dgomes so that this PR can be pushed into main line code?

Reply:
> if should default to the previously configured value, no ?

That's probably how it should work, yes. I'll look into changing that.

Reply:
bump for this PR @dgomes  and @rdfurman

Reply:
bump @rdfurman any chance you can get these changes pushed?